### PR TITLE
AnonymousAuthorizationToken fix and get_auth_token_age update

### PR DESCRIPTION
**Pull request by Formulka**

### CODE REVIEW

File: auth_token/middleware.py [hunk 1]
```diff
@@ -41,7 +41,8 @@ def process_request(self, request):
 
     def _update_token_and_cookie(self, request, response, max_age, expires):
         request.token.change_and_save(
-            expires_at=compute_authorization_token_expires_at(get_auth_token_age(request)),
+            expires_at=compute_authorization_token_expires_at(get_auth_token_age(
+                request, request.token.user, request.token.auth_slug)),
```
Comment (Formulka) [inline auth_token/middleware.py:45]: the problem is this is User model

File: auth_token/utils.py [hunk 2]
```diff
@@ -188,7 +188,7 @@ def login(request, user, auth_slug=None, related_objs=None, backend=None, allowe
         allowed_header=allowed_header,
         is_authenticated=not two_factor_login,
         expires_at=compute_authorization_token_expires_at(
-            expiration if expiration is not None else get_auth_token_age(request)),
+            expiration if expiration is not None else get_auth_token_age(request, user, auth_slug)),
```
Comment (Formulka) [inline auth_token/utils.py:191]: and this is subclass of the User model

File: example/dj/apps/app/tests/utils.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def test_otp_generator():
     return '12345'
 
 
-def auth_token_age_test(request):
+def auth_token_age_test(request, user, auth_token):
```
Comment (Formulka) [inline example/dj/apps/app/tests/utils.py:52]: so here you get either User or subclass of User, imho my original solution was better, let it fall through to the default and let the middleware update the expiration

File: auth_token/middleware.py (hunk 1 above)
Comment (Formulka) [reply auth_token/middleware.py:45]: I can't use reques.user, because that can be AnonymousUser and wouldn't solve the issue I'm trying to solve here

Comment (matllubos) [reply auth_token/middleware.py:45]: yes this is ok for me

File: auth_token/utils.py (hunk 2 above)
Comment (matllubos) [reply auth_token/utils.py:191]: only for our project..

Comment (Formulka) [reply auth_token/utils.py:191]: good point

File: example/dj/apps/app/tests/utils.py (hunk 3 above)
Comment (Formulka) [reply example/dj/apps/app/tests/utils.py:52]: nevermind, it is an edge case
